src/config.ts

```diff
@@ -405,23 +405,23 @@ export const GAS_SCHEDULE_V1: FeeSchedule = {
     [ERC20BridgeSource.MultiBridge]: () => 3.5e5,
     [ERC20BridgeSource.UniswapV2]: fillData => {
         // TODO: Different base cost if to/from ETH.
-        let gas = 100e3;
+        let gas = 90e3;
         const path = (fillData as UniswapV2FillData).tokenAddressPath;
         if (path.length > 2) {
-            gas += (path.length - 2) * 50e3; // +50k for each hop.
+            gas += (path.length - 2) * 60e3; // +60k for each hop.
```

Comment:
50k per hop seemed a bit low, see this UniV2 [DAI-WETH-YFI trade](https://dashboard.tenderly.co/0x-org/exchange-proxy/tx/main/0x23e34da86eeb2d13a06ac84887364bf77c082f0477ce5e82fccb023e6696cbde/gas-usage)
or this Sushi [UNI-WETH-YFI trade](https://dashboard.tenderly.co/0x-org/exchange-proxy/tx/main/0x8fc6862e5a255893b87df1fc7114ea17b735397d285ee9c871fe628ff60383d4/gas-usage)